src/main/java/com/atlassian/shele/shelePlugin/ao/ProjectEntityService.java

```diff
@@ -22,15 +22,17 @@ public ProjectEntityService(@ComponentImport ActiveObjects activeObjects, Projec
 
     public List<ProjectDTO> saveEntity(ProjectDTO dto) {
         List<ProjectEntity> projectEntityList = new ArrayList<>();
-        List<String> list = dto.getProject();
-        List<Long> longs = dto.getEventTypeIds();
-        for (String s : list) {
+        List<String> stringList = dto.getProject();
```

Comment:
have you read this documents [Oracle](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) and [Google](https://google.github.io/styleguide/javaguide.html) code conventions?

it should be something like `List<String> projects` or `List<String> projectKeys`
the same is for `List<Long> eventTypeIds`